fv3core/grid/__init__.py

```diff
@@ -0,0 +1,35 @@
+# flake8: noqa: F401
+
+from .eta import set_eta
+from .generation import MetricTerms
+from .geometry import (
+    calc_unit_vector_south,
+    calc_unit_vector_west,
+    calculate_divg_del6,
+    calculate_grid_a,
+    calculate_grid_z,
+    calculate_l2c_vu,
+    calculate_supergrid_cos_sin,
+    calculate_trig_uv,
+    edge_factors,
+    efactor_a2c_v,
+    generate_xy_unit_vectors,
+    get_center_vector,
+    supergrid_corner_fix,
+    unit_vector_lonlat,
+)
+from .gnomonic import (
+    get_area,
+    global_gnomonic_ed,
+    gnomonic_grid,
+    great_circle_distance_along_axis,
+    local_gnomonic_ed,
+    lon_lat_corner_to_cell_center,
+    lon_lat_midpoint,
+    lon_lat_to_xyz,
+    set_c_grid_tile_border_area,
+    set_corner_area_to_triangle_area,
+    set_tile_border_dxc,
+    set_tile_border_dyc,
+)
+from .mirror import global_mirror_grid, mirror_grid, set_halo_nan
```

Comment:
Do you mean for all of these to be public API? If not, can you use private namespacing to the ones you expect to be private? This affects how complete the docstrings need to be.